### PR TITLE
Jetpack Onboarding: Update illustrations in JPO flow

### PR DESCRIPTION
**Pull request by joanrho**
This PR updates three illustration assets in three of the steps of the Jetpack Onboarding (JPO), updates the filenames and paths in the respective steps jsx files, and tweaks the styling a little bit. The `jetpack-start.svg` (formerly known as `site-title.svg`) will also be used in our updated Plans view (cc @AnnaMag who is working on that)

JPO Site title step (uses `jetpack-start.svg`)
screen shot 2018-03-02 at 12 14 55 pm

JPO Business address step (uses `jetpack-business-address.svg`)
screen shot 2018-03-02 at 12 15 01 pm

JPO Stats step (uses `jetpack-stats.svg`)
screen shot 2018-03-02 at 12 15 06 pm

**Testing instructions:**
* Checkout this branch
* Pick a Jetpack site.
* Head to /wp-admin/admin.php?page=jetpack&action=onboard&calypso_env=development on the Jetpack site to start the onboarding flow.
* On the site title step, verify the illustration looks the same as the above "After" shot.
* In the site type step, select "Business".
* Skip to the business address step and verify the illustration looks the same as the above "After" shot.
* Skip to the stats step and verify the illustration looks the same as the above "After" shot.

### CODE REVIEW

File: client/jetpack-onboarding/steps/business-address.jsx [hunk 1]
```diff
@@ -157,7 +157,7 @@ class JetpackOnboardingBusinessAddressStep extends React.PureComponent {
 				buttonLabel={ translate( 'Add a business address' ) }
 				e2eType="business-address"
 				header={ this.renderHeader() }
-				illustration="/calypso/images/illustrations/business-address.svg"
+				illustration="/calypso/images/illustrations/illustration-businessaddress-jetpack.svg"
```
Comment (tyxla) [inline client/jetpack-onboarding/steps/business-address.jsx]: I'd name this one `jetpack-business-address.svg` instead, or even  (`business-address.svg` - we introduced this illustration specifically for this step, so we can replace it).

File: client/jetpack-onboarding/steps/site-title.jsx [hunk 2]
```diff
@@ -79,7 +79,7 @@ class JetpackOnboardingSiteTitleStep extends React.PureComponent {
 				<Card className="steps__form">
 					<img
 						className="steps__illustration"
-						src={ '/calypso/images/illustrations/site-title.svg' }
+						src={ '/calypso/images/illustrations/illustration-start-jetpack.svg' }
```
Comment (tyxla) [inline client/jetpack-onboarding/steps/site-title.jsx]: I'd name this one `jetpack-start.svg` (we introduced the `jetpack-start` illustration specifically for this step, so we can just replace the file)

File: client/jetpack-onboarding/steps/stats.jsx [hunk 3]
```diff
@@ -79,7 +79,7 @@ class JetpackOnboardingStatsStep extends React.Component {
 				buttonLabel={ translate( 'Activate stats' ) }
 				e2eType="activate-stats"
 				header={ header }
-				illustration="/calypso/images/illustrations/jetpack-stats.svg"
+				illustration="/calypso/images/illustrations/illustration-stats-jetpack.svg"
```
Comment (tyxla) [inline client/jetpack-onboarding/steps/stats.jsx]: I'd name this one `jetpack-stats.svg` instead (we introduced it for this step, so we can just replace the file).

File: client/jetpack-onboarding/steps/business-address.jsx (hunk 1 above)
Comment (joanrho) [reply client/jetpack-onboarding/steps/business-address.jsx]: 👍

File: client/jetpack-onboarding/steps/site-title.jsx (hunk 2 above)
Comment (joanrho) [reply client/jetpack-onboarding/steps/site-title.jsx]: 👍

File: client/jetpack-onboarding/steps/stats.jsx (hunk 3 above)
Comment (joanrho) [reply client/jetpack-onboarding/steps/stats.jsx]: 👍

**Comment by joanrho** [comment]
@tyxla thanks for the feedback. Filenames and paths all updated. Want to take another spin before I commit?

**Comment by tyxla** [comment]
@joanrho seems like some illustrations (for example the ones that start with `illustration-`) were added, but they are not used at all? Perhaps you'd want to delete them from Calypso?

**Pull request merged**